client/post-editor/editor-ground-control/style.scss

```diff
@@ -38,15 +47,36 @@
 	margin: 0 4px 8px;
 }
 
-.editor-ground-control__publish-button {
+.editor-ground-control .editor-publish-button {
```

Comment:
Any reason we shouldn't move this to `client/post-editor/editor-publish-button/style.scss` as the base `.editor-publish-button` style ?